vignettes/case-study-scaling.Rmd

```diff
@@ -0,0 +1,222 @@
+---
+title: "Case study: Scaling an app"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+`shinyloadtest` and `shinycannon` are tools to help you assess your application's performance. Since they are only tools to measure performance and not to improve it, they must be used as part of a larger workflow.
+
+In this case study, we'll introduce a workflow — *the optimization loop* — and apply the workflow to the process of optimizing the [cranwhales](https://github.com/rstudio/cranwhales) demo application.
+
+For a lighter treatment of `shinyloadtest` that will get you using it faster, you may consider starting with [the introductory tutorial](..) instead.
+
+## The Optimization Loop
+
+![](../docs/inst/img/loop.svg)
+
+
+
+The optimization loop is a general methodology for approaching optimization work. Applied to the problem of capacity optimization, which is the area `shinyloadtest` is helpful in, the steps are defined as:
+
+1. **Benchmark**: Pick a number *n*, how many users your application must concurrently support. Simulate *n* users interacting with your application and collect latency metrics. Use `shinyloadtest::record_session` to record a realistic interaction, and the `shinycannon` tool to simulate *n* users performing the recorded interaction.
```

Comment:
Maybe add links to each vignette at the end of each item? (once we get the site in place)